bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/templates/TemplateListSelectionPage.java

```diff
@@ -134,6 +188,8 @@ public void setInitialTemplateId(ITemplateNode templateNode)
 	@Override
 	public void setVisible(boolean visible)
 	{
+
+		projectNameField.setText(prevPage.getProjectName());
```

Comment:
Can we set this in the **createProjectNameGroup**() itself as this is a one-time activity, and importantly this call is inappropriate for the **setVisible**() method